src/app/page.jsx

```diff
@@ -1,6 +1,22 @@
+'use client'
```

Comment:
- Changed this file from Typescript to Javascript
- Changed this component to a client component by using `use client` since it's now using the `useEffect` hook